src/pages/profile/profile.html

```diff
@@ -0,0 +1,54 @@
+<!--
+  Generated template for the ProfilePage page.
+
+  See http://ionicframework.com/docs/components/#navigation for more info on
+  Ionic pages and navigation.
+-->
+<ion-header>
+
+  <ion-navbar>
+    <ion-buttons left>
+      <button ion-button icon-only color="secondarydark" (click)="goSettings()">
+        <ion-icon name="cog"></ion-icon>
+      </button>
+    </ion-buttons>
+    <ion-title>Profile</ion-title>
+  </ion-navbar>
+
+</ion-header>
+
+
+<ion-content>
+  <!-- Avatar -->
+  <ion-card no-margin padding-vertical>
+    <img src="assets/imgs/profile.png" />
+    <ion-card-content margin-top no-padding>
+      <ion-card-title>
+        Modus Create
+      </ion-card-title>
+    </ion-card-content>
+  </ion-card>
+
+
+  <!-- Tabs -->
+  <div ion-row text-center class="profile-tab-buttons" #tabs>
+    <div ion-col no-padding>
+      <button type="button" ion-button clear [ngClass]="{ 'active': selectedTab === profileFilters.todo }" (click)="toogle(profileFilters.todo)">
+        ToDos
+      </button>
+    </div>
+    <div ion-col no-padding>
+      <button type="button" ion-button clear [ngClass]="{ 'active': selectedTab === profileFilters.note }" (click)="toogle(profileFilters.note)">
+        Notes
+      </button>
+    </div>
+  </div>
+
+  <!-- List -->
+  <ion-scroll scrollY="true" [ngStyle]="{ 'height.px': scrollHeight }">
+    <ion-list padding no-lines>
+      <list-item *ngFor="let item of (items$ | async)" [item]="item"></list-item>
```

Comment:
Always a good idea to use `trackBy` function with the ngFor stuff for better performance